gravitee-am-management-api/gravitee-am-management-api-rest/src/test/java/io/gravitee/am/management/handlers/management/api/resources/ApplicationResourceTest.java

```diff
@@ -381,4 +374,13 @@ private Application buildApplicationMock() {
 
         return mockApplication;
     }
+
+    private SortedSet<ApplicationIdentityProvider> getApplicationIdentityProviders() {
+        var patchAppIdp = new ApplicationIdentityProvider("id1", 1);
+        var patchAppIdp2 = new ApplicationIdentityProvider("id2", 2);
+        var set = new TreeSet<ApplicationIdentityProvider>();
+        set.add(patchAppIdp);
+        set.add(patchAppIdp2);
+        return set;
```

Comment:
You could do this in one statement:

`
        return new TreeSet<>(Set.of(new ApplicationIdentityProvider("id1", 1), new ApplicationIdentityProvider("id2", 1)));`